IngameNetManager.cs

```diff
@@ -2192,25 +2208,38 @@ public void DestroyActor(Actor actor)
         }
         
         /// <summary>
-        /// Change Temporarily the animator controller for one that contains the kick animation
+        /// Change Temporarily the animator controller for one that contains the kick animation and reproduce a sound
         /// </summary>
         private IEnumerator PerformKick(Actor actor)
         {
-            if (kickController == null)
+            if (KickController == null)
             {
                 Plugin.logger.LogError("Kick Animation Failed!");
                 yield break;
             }
-            
             var actorAnimator = actor.animator;
-            var preKickController = actorAnimator.runtimeAnimatorController;
-
-            actorAnimator.runtimeAnimatorController = kickController;
+            
+            if (OldKickController == null)
+            {
+                OldKickController = actorAnimator.runtimeAnimatorController;
+            }
+            
+            actorAnimator.runtimeAnimatorController = KickController;
             actorAnimator.SetTrigger("kick");
 
+            // Create a new object in the scene and play the kickSound in the position of the actor
```

Comment:
BTW Unity has something that will do this for you. It has the added benefit of spawning/despawning the object for each instance so in the edge case that two people kick at the same time it won't move the audio.
https://docs.unity3d.com/ScriptReference/AudioSource.PlayClipAtPoint.html

Reply:
I tried to use this Audio Source Method, but It doesnt seem to exist in the unity version RF is made. Because I cant find it

![AudioSource](https://user-images.githubusercontent.com/91845721/182434623-ca1e3fb2-c0db-4129-bbb3-afa888cf4715.png)

Reply:
It's static, so you can't use it on an object's component. You could do something like
```cs
AudioSource.PlayClipAtPoint(KickSound, actor.transform.position);
```
You won't need the KickSoundSource at all this way.

Reply:
Thanks for letting me now. Is it better to move the Coroutine to the KickPacket script? Just to keep the `IngameNetManager` cleaner

Reply:
Sure, that would be nice. I'm thinking that can be done in a later PR, though, since this looks good to merge.

Reply:
Sure thing